ast_canopy/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_CXX_FLAGS_DEBUG  "-g")
 
 project(${SKBUILD_PROJECT_NAME}
         VERSION ${SKBUILD_PROJECT_VERSION}
-        LANGUAGES CXX)
+	LANGUAGES C CXX)
```

Comment:
A comment here mentioning "C" language is needed to build in pytorch image on an aarch64 machine.